### PR TITLE
Add MATEKF411SE Target

### PR DESCRIPTION
**Pull request by MATEKSYS**
MATEKF411SE Target is for FC F411-WSE
Rearrange the pinout based on MATEKF411 target. it supports 2x softserial_tx, 4xADC, 6xPWM and LED working together.

### CODE REVIEW

**Comment by Jetrell** [comment]
@MATEKSYS Would you mind informing us of the pin locations on the boards, with this new target?
 This maybe helpful for a future reference.

**Comment by Jetrell** [comment]
@MATEKSYS Would you mind informing us of where these extra I/O's will be located on both boards, with this newly defined target?
This maybe helpful for a quick future reference.

**Comment by MATEKSYS** [comment]
Difference between F411 and F411SE
F411vsF411SE

File: src/main/target/MATEKF411SE/target.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#include "drivers/bus.h"
+#include "drivers/io.h"
+#include "drivers/pwm_mapping.h"
+#include "drivers/timer.h"
+#include "drivers/pinio.h"
+
+const timerHardware_t timerHardware[] = {
+    DEF_TIM(TIM3, CH1, PB4,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR,  0, 0), // S1  D(1,4,5)
+    DEF_TIM(TIM3, CH2, PB5,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR,  0, 0), // S2  D(1,5,5)
+    DEF_TIM(TIM1, CH1, PA8,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 0), // S3  D(2,1,6)
```
Comment (MATEKSYS) [inline src/main/target/MATEKF411SE/target.c]: DEF_TIM(TIM1, CH1, PA8,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S3  D(2,1,6)

File: src/main/target/MATEKF411SE/target.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    DEF_TIM(TIM1, CH2, PA9,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 0), // S4  D(2,2,6)
```
Comment (MATEKSYS) [inline src/main/target/MATEKF411SE/target.c]: DEF_TIM(TIM1, CH2, PA9,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S4  D(2,2,6)

File: src/main/target/MATEKF411SE/target.c [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    DEF_TIM(TIM1, CH3, PA10, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 0), // S5  D(2,6,6)
```
Comment (MATEKSYS) [inline src/main/target/MATEKF411SE/target.c]: DEF_TIM(TIM1, CH3, PA10, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S5  D(2,6,6)

Comment (MATEKSYS) [inline src/main/target/MATEKF411SE/target.c]: DEF_TIM(TIM1, CH1, PA8,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S3  D(2,1,6)
    DEF_TIM(TIM1, CH2, PA9,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S4  D(2,2,6)
    DEF_TIM(TIM1, CH3, PA10, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S5  D(2,6,6)

Comment (MATEKSYS) [inline src/main/target/MATEKF411SE/target.c]: DEF_TIM(TIM1, CH1, PA8,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S3  D(2,1,6)
    DEF_TIM(TIM1, CH2, PA9,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S4  D(2,2,6)
    DEF_TIM(TIM1, CH3, PA10, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,  0, 1), // S5  D(2,6,6)

**Review by MATEKSYS — COMMENTED** [review]
Fix DMA

**Comment by Jetrell** [comment]
@MATEKSYS Can I make some suggestions.
I see the extra innovations that this board has over the F411-Wing. Filtered Vbat, Cam switcher.
But it still lacks one thing that hasn't been improved. And as a multiple user of the F411-Wing, I and many of us, wished it had at least 1 extra UART.

The issue I see with your current F411-WSE pin allocation. Is that it shares Softserial2 and UART 2.
I know i speak for others as well, when I say that it would be better to place SoftSerial 2 on the LED or Buzzer pins.
Plus.... I think it would be better to have the ability for full duplex soft serial on ST1 and LED pins. That would go a long way towards making it more like a third UART. It could be used for HD camera controls etc.

**Comment by Jetrell** [comment]
@MATEKSYS Can I make some suggestions for the F411-WSE
I see the extra innovations that this board has over the F411-Wing. Filtered Vbat, Cam switcher.
But it still lacks one thing that hasn't been improved. And as a multiple user of the F411-Wing, I and many of us, wished it had at least 1 extra UART.

The issue I see with your current pin allocation for the F411-WSE. Is that it shares Softserial2 and UART 2.
I know i speak for others as well, when I say that it would be better to place SoftSerial 2 on the LED or Buzzer pins.
Plus.... I think it would be better to have the ability for full duplex soft serial on ST1 and LED pins. That would go a long way towards making it more like a third UART. It could be used for HD camera controls etc.

**Comment by DzikuVx** [comment]
I'm restarting Travis lets see if it will fail this time or not. I think it's time to merge it

**Comment by DzikuVx** [comment]
@digitalentity @MATEKSYS shall we merge this?

**Comment by MATEKSYS** [comment]
@DzikuVx  Pls merge it. thanks

**Comment by DzikuVx** [comment]
and merged, thanks a lot

**Pull request merged**